docs/content/integrations/airflow/from-airflow-to-dagster.mdx

```diff
@@ -0,0 +1,260 @@
+---
+title: "Learning Dagster from Airlfow"
+description: How to get started with Dagster from an Airflow background
+---
+
+# Learning Dagster from Airflow
+
+Airflow users often come to Dagster for a variety of reasons. Some want to take advantage of our great story around testability. Others want to deploy to an orchestrator that is container native and has better support for multiple Python environments. And still others are attracted to the promise of a better abstraction: software-defined assets.
+
+In this tutorial, we'll help you make the switch from Airflow to Dagster. Map the concepts you're familiar with in Airflow to Dagster and provide some examples of the work you did in Airflow can be done in Dagster.
+
+By the end of this tutorial, you will:
+
+- Understand how concepts in Airflow map onto Dagster
+- Understand how to author a Dagster Pipeline similiar to a Scheduled Airflow DAG
```

Comment:
There will be a single point of this tutorial: we will define the DAG from the [Airflow tutorial](https://airflow.apache.org/docs/apache-airflow/stable/tutorial/fundamentals.html) as a Dagster job.